appveyor.yml

```diff
@@ -18,7 +18,7 @@ install:
 - ps: |
     $env:CommitHash = "$env:APPVEYOR_REPO_COMMIT"
 
-    .\dotnet-install.ps1 -Version 2.1.300 -Architecture x86
+    .\dotnet-install.ps1 -Version 3.1.410 -Architecture x86
```

Comment:
Bumping to 3.1 which is required to use the new C# 8.0 Nullable Reference Types language feature I've used in a few of the public interfaces. https://docs.microsoft.com/en-us/dotnet/csharp/nullable-references